README.md

```diff
@@ -22,17 +25,42 @@ for client libraries to work.
         --ddl-file src/main/java/com/google/finapp/schema.sdl
     $ export SPANNER_EMULATOR_HOST="localhost:9010"
     ```
+   In a **separate terminal window**, if you are **not** using the emulator for the first time, run:
+   ```
+    $ gcloud config configurations activate emulator
+    $ gcloud config set auth/disable_credentials true
+    $ gcloud config set project your-project-id
+    $ gcloud config set api_endpoint_overrides/spanner http://localhost:9020/
+    $ gcloud spanner instances create test-instance \
+        --config=emulator-config --description="Test Instance" --nodes=1
+    $ gcloud config set spanner/instance test-instance
+    $ gcloud spanner databases create test-database \
+        --ddl-file src/main/java/com/google/finapp/schema.sdl
+    $ export SPANNER_EMULATOR_HOST="localhost:9010"
+    ```
```

Comment:
A lot of this is repeated--if I'm correct, everything except the first line. Maybe instead say something like:

 In a **separate terminal window**, if you are using the emulator **for the first time**, run:
`    $ gcloud config configurations create emulator`

If you are **not** using the emulator for the first time, run:
`    $ gcloud config configurations activate emulator`

Then run: 
_everything else_